lib/registry.js

```diff
@@ -247,6 +249,30 @@ Registry.prototype.configureModel = function(ModelCtor, config) {
       'Use `null` or `false` to mark models not attached to any data source.',
       modelName);
   }
+
+  // Remote methods
+  this._defineRemoteMethods(ModelCtor, config.methods);
+};
+
+Registry.prototype._defineRemoteMethods = function(ModelCtor, methods) {
+  if (!methods) return;
+  if (typeof methods !== 'object') {
+    console.warn('Ignoring non-object "methods" setting of "%s".',
+      ModelCtor.modelName);
+    return;
+  }
+
+  Object.keys(methods).forEach(function(key) {
+    var meta = methods[key];
+    if (typeof meta.isStatic !== 'boolean') {
+      console.warn('Remoting metadata for "%s.%s" is missing "isStatic" ' +
+        'flag, the method is registered as an instance method.',
+        ModelCtor.modelName,
+        key);
+      console.warn('This behaviour may change in the next major version.');
```

Comment:
I decided to report a warning instead of throwing an exception, because what if somebody was already using the "methods" option and implemented a solution similar to this patch? Throwing an exception would break backwards-compatibility for that case.

However, if you @ritch think I am too cautious and it's ok to throw an error, then I'll rework this part.